service-01/Request-ID.asn1

```diff
@@ -24,12 +24,12 @@
 Request-ID DEFINITIONS AUTOMATIC TAGS ::= BEGIN
 EXPORTS ALL;
 IMPORTS
-    PacketVersionNumber, PacketSequenceControl FROM PacketTypes
+    PacketVersionNumberValue, PacketSequenceControl FROM PacketTypes
```

Comment:
why `Value` suffix?

Reply:
Wanted to somehow distinguish variable and constant type definitions

Reply:
drop it - important thing is "what this thing is in ASN.1" and constants are "ACN thing". In theory all those ASN.1 models could be used without ACN. So while naming etc. focus on ASN.1 and treat ACN as utility